external-crates/move/crates/move-compiler/src/naming/translate.rs

```diff
@@ -470,6 +470,10 @@ impl<'env> Context<'env> {
     }
 
     fn enter_nominal_block(&mut self, name: Option<P::BlockLabel>, name_type: NominalBlockType) {
+        debug_assert!(
+            self.nominal_blocks.len() < 100,
+            "Nominal block list exceeded 100."
```

Comment:
Maybe just a small comment about us needing to reevaluate the data structure here. Otherwise, thanks!

Reply:
I added the comment a few lines up, at the context definion.